### PR TITLE
Some untranslated UI strings

### PR DESCRIPTION
**Pull request by goodyttoor**

### CODE REVIEW

File: editor/editor_data.cpp [hunk 1]
```diff
@@ -556,7 +556,7 @@ bool EditorData::check_and_update_scene(int p_idx) {
 
 	bool must_reload = _find_updated_instances(edited_scene[p_idx].root, edited_scene[p_idx].root, checked_scenes);
 
-	print_line("MUST RELOAD? " + itos(must_reload));
+	print_line(TTR("MUST RELOAD? ") + itos(must_reload));
```
Comment (Calinou) [inline editor/editor_data.cpp]: Debugging strings don't need to be translated – they should be removed instead. This should probably be done in a separate pull request.

File: editor/editor_audio_buses.cpp [hunk 2]
```diff
@@ -461,10 +461,10 @@ void EditorAudioBus::drop_data(const Point2 &p_point, const Variant &p_data) {
 
 Variant EditorAudioBus::get_drag_data_fw(const Point2 &p_point, Control *p_from) {
 
-	print_line("drag fw");
+	print_line(TTR("drag fw"));
```
Comment (Calinou) [inline editor/editor_audio_buses.cpp]: Debugging strings don't need to be translated – they should be removed instead. This should probably be done in a separate pull request.

File: editor/editor_audio_buses.cpp [hunk 3]
```diff
@@ -461,10 +461,10 @@ void EditorAudioBus::drop_data(const Point2 &p_point, const Variant &p_data) {
...
 	TreeItem *item = effects->get_item_at_position(p_point);
 	if (!item) {
-		print_line("no item");
+		print_line(TTR("no item"));
```
Comment (Calinou) [inline editor/editor_audio_buses.cpp]: Debugging strings don't need to be translated – they should be removed instead. This should probably be done in a separate pull request.

File: editor/export_template_manager.cpp [hunk 4]
```diff
@@ -499,7 +499,7 @@ void ExportTemplateManager::_notification(int p_what) {
 
 	if (p_what == NOTIFICATION_VISIBILITY_CHANGED) {
 		if (!is_visible_in_tree()) {
-			print_line("closed");
+			print_line(TTR("closed"));
```
Comment (Calinou) [inline editor/export_template_manager.cpp]: Debugging strings don't need to be translated – they should be removed instead. This should probably be done in a separate pull request.

File: editor/fileserver/editor_file_server.cpp [hunk 5]
```diff
@@ -258,7 +258,7 @@ void EditorFileServer::_subthread_start(void *s) {
 			} break;
 			case FileAccessNetwork::COMMAND_CLOSE: {
 
-				print_line("CLOSED");
+				print_line(TTR("CLOSED"));
```
Comment (Calinou) [inline editor/fileserver/editor_file_server.cpp]: Debugging strings don't need to be translated – they should be removed instead. This should probably be done in a separate pull request.

**Review by Calinou — COMMENTED** [review]
See the commented lines – debugging strings shouldn't be translated (as most of them should be removed, or at least hidden by default). Removal of debugging messages can be done in a separate pull request.

**Comment by goodyttoor** [comment]
Oops sorry first time here. Remove those thank you.

**Comment by goodyttoor** [comment]
I made a mess on my branch. Let me make a new PR

**Comment by goodyttoor** [comment]
Make a mess on my branch. Git noop here :P

I will make new PR sorry
